scss/dialogs/_qr-code.scss

```diff
@@ -27,7 +27,7 @@
 
 .scan-qr-description {
   position: relative;
-  z-index: 10;
+  z-index: 1;
```

Comment:
Same here. This is also a locally scoped z-index?

Reply:
no it isn't not really necessary with those z-index 1 values, but sure for completeness sake I can do it.